docs/content/Caching/Getting-Started-Pre-Aggregations.md

```diff
@@ -159,7 +159,175 @@ the pre-aggregations are rebuilt.
 These refreshes can be done on-demand, or [in the background as a scheduled
 process][ref-caching-preaggs-bk-refresh].
 
+## Ensuring pre-aggregations are targeted by queries
```

Comment:
@hassankhan You've done a really great job explaining the concepts. My only major suggestion here would be to make the end result more accessible and actionable for developers, so they don't have to read through the whole subsection to get the idea. Here's how I think it could be done:

* We can prepend the conceptual topics with an intro with a set of "simple rules" to consider, essentially moving some of the considerations and contents of the final image here
* It may be as simple as saying something like: 1. Cube will look if all measures, dimensions, and time dimensions with their granularities matched. If so, success, the pre-agg is used. 3. If this is off (like, you use `countDistinctApprox`), it's because of non-additivity, see below 2. If this is off, it's because of this leaf thing, see below 4. ...
* So, in the end of the intro the developers would either fix their problem by checking it against these simple rules or know for sure they need to dive deeper
* Here's where the rest comes handy

Reply:
Part of the reason why I structured it this way was to match the earlier part of the same document; the intended target audience here is someone who is brand-new to pre-aggregations so we wanted to try and explain the concepts first before getting into the nitty-gritty.

I'm not against adding a shorter version of the flow, but I couldn't think of a good enough way to summarize it; the diagram is great but it is quite large and feels a bit much as the first thing.

I'll have another go at it 👍